<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowMatch</class>
 <widget class="QMainWindow" name="WindowMatch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1151</width>
      <height>611</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 127)</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>gestion des matches</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>id</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>idstade</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>56</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>date</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>56</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>sport</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>56</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>nbrp</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ajouter">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>150</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



color: white;



background:rgb(0, 170, 255) ;



font: bold, &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>ajouter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



background: white;



selection-background-color: gray;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_idstade">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



background: white;



selection-background-color: gray;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_date">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



background: white;



selection-background-color: gray;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_sport">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>100</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



background: white;



selection-background-color: gray;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nbrp">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>130</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



background: white;



selection-background-color: gray;</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>341</width>
        <height>221</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>280</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



color: white;



background:rgb(0, 170, 255) ;



font: bold, &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>stat</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pdf">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>470</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



color: white;



background:rgb(0, 170, 255) ;



font: bold, &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>pdf</string>
      </property>
     </widget>
     <widget class="QPushButton" name="triID">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>380</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



color: white;



background:rgb(0, 170, 255) ;



font: bold, &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>triID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="triIDSTADE">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>330</y>
        <width>151</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



color: white;



background:rgb(0, 170, 255) ;



font: bold, &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>triIDSTADE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="recherche">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>480</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



color: white;



background:rgb(0, 170, 255) ;



font: bold, &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>recherche</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_rech">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>480</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



background: white;



selection-background-color: gray;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="imprimer">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>430</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



color: white;



background:rgb(0, 170, 255) ;



font: bold, &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>imprimer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>60</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>id</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ids">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>60</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



background: white;



selection-background-color: gray;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="supprimer">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>100</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



color: white;



background:rgb(0, 170, 255) ;



font: bold, &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>60</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>id</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>120</y>
        <width>56</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>date</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>90</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>idstade</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>150</y>
        <width>56</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>sport</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>180</y>
        <width>56</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>nbrp</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nbrpm">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>190</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



background: white;



selection-background-color: gray;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modifier">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>260</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



color: white;



background:rgb(0, 170, 255) ;



font: bold, &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>modifier</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_datem">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>130</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



background: white;



selection-background-color: gray;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_idm">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>60</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



background: white;



selection-background-color: gray;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_sportm">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>160</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



background: white;



selection-background-color: gray;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_idstadem">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>90</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



background: white;



selection-background-color: gray;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>200</y>
        <width>181</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



color: white;



background:rgb(0, 170, 255) ;



font: bold, &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>fermeture manuelle de toit </string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>sms</string>
     </attribute>
     <widget class="QPushButton" name="sms">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>150</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



color: white;



background:rgb(0, 170, 255) ;



font: bold, &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>sms</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>160</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;



border-radius: 10px;



padding: 0 8px;



background: white;



selection-background-color: gray;</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>190</y>
        <width>101</width>
        <height>91</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
